metricflow-semantics/tests_metricflow_semantics/snapshots/test_messages.py/str/test_long_invalid_metric__result.txt

```diff
@@ -16,7 +16,3 @@ Error #1:
   Query Input:
 
     long_invalid_metric_name__________________________________________________
-
```

Comment:
These are removed now since metric names aren't resolved at this point, but the above should be enough